ui/src/app/services/chrome-recoder.service.ts

```diff
@@ -75,7 +75,7 @@ export class ChromeRecorderService {
       this.messageEvent = event;
       if (this.isInstalledEvent) {
         this.isInstalled = true;
-      } else if (this.isStoppedRecorder) {
+      } else if (this.isStoppedRecorder && this.isStepRecording) {
```

Comment:
how does the isStoppedRecorder and isStepRecording align same?

Reply:
the prop can be named as isStepRecorder | isElementRecorder accordingly based on the context of recorder.

Reply:
updated to isSteprecorder